packages/builders/scripts/fast-usdc/init-fast-usdc.js

```diff
@@ -0,0 +1,101 @@
+// @ts-check
+import { makeHelpers } from '@agoric/deploy-script-support';
+import { AmountMath } from '@agoric/ertp';
+import {
+  FastUSDCConfigShape,
+  getManifestForFastUSDC,
+} from '@agoric/fast-usdc/src/fast-usdc.start.js';
+import { toExternalConfig } from '@agoric/fast-usdc/src/utils/config-marshal.js';
+import { objectMap } from '@agoric/internal';
+import {
+  multiplyBy,
+  parseRatio,
+} from '@agoric/zoe/src/contractSupport/ratio.js';
+import { Far } from '@endo/far';
+import { parseArgs } from 'node:util';
+
+/**
+ * @import {CoreEvalBuilder} from '@agoric/deploy-script-support/src/externalTypes.js'
+ * @import {ParseArgsConfig} from 'node:util'
+ * @import {FastUSDCConfig} from '@agoric/fast-usdc/src/fast-usdc.start.js'
+ */
+
+/** @type {ParseArgsConfig['options']} */
+const optionsConfig = {
+  contractFee: { type: 'string', default: '0.01' },
+  poolFee: { type: 'string', default: '0.01' },
+  oracle: { type: 'string', multiple: true },
+};
+const oraclesRequiredUsage = 'use --oracle name:address ...';
+/**
+ * @typedef {{
+ *   contractFee: string;
+ *   poolFee: string;
+ *   oracle?: string[];
+ * }} FastUSDCOpts
+ */
+
+const crossVatContext = /** @type {const} */ ({
+  /** @type {Brand<'nat'>} */
+  USDC: Far('USDC Brand'),
+});
+const { USDC } = crossVatContext;
+const USDC_DECIMALS = 6;
+const unit = AmountMath.make(USDC, 10n ** BigInt(USDC_DECIMALS));
+
+/** @type {CoreEvalBuilder} */
+export const defaultProposalBuilder = async (
+  { publishRef, install },
+  /** @type {FastUSDCConfig} */ config,
+) => {
+  return harden({
+    sourceSpec: '@agoric/fast-usdc/src/fast-usdc.start.js',
+    /** @type {[string, Parameters<typeof getManifestForFastUSDC>[1]]} */
+    getManifestCall: [
+      getManifestForFastUSDC.name,
+      {
+        options: toExternalConfig(config, crossVatContext, FastUSDCConfigShape),
+        installKeys: {
+          fastUSDC: publishRef(
+            install('@agoric/fast-usdc/src/fast-usdc.contract.js'),
+          ),
+        },
+      },
+    ],
+  });
+};
+
+export default async (homeP, endowments) => {
```

Comment:
please annotate to get docs for `scriptArgs`
```suggestion
/** @type {DeployScriptFunction} */
export default async (homeP, endowments) => {
```